src/time_unix.c

```diff
@@ -30,7 +30,11 @@ extern "C"
 #include <math.h>
 
 #if defined(__ZEPHYR__)
+#if KERNEL_VERSION_MAJOR >= 3
```

Comment:
This definition is only provided when building with https://github.com/micro-ROS/micro_ros_zephyr_module right? Is it not Zephyr standard?

Reply:
@pablogs9 , you are right, there is a "zephyrzed" way of handling this, please check the new one.